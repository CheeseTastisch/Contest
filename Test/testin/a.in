1 2 3
15.7 18.9
and some things