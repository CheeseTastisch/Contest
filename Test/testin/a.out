3
3 34.599999999999994 and some-things

